tests/include/unit_test.hpp

```diff
@@ -1,15 +1,6 @@
-
-#if __GNUC__ >= 8 && defined( __has_warning )
-#   if __has_warning( "-Wformat-overflow=" )
-#       define SUPPRESSING
-#       pragma GCC diagnostic push
-#       pragma GCC diagnostic ignored "-Wformat-overflow="
-#   endif
-#endif
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wformat-overflow="
 
```

Comment:
clang compatibility will be broken as clang does not recognize format-overflow. Check solution here:
https://github.com/abilitiinc/AbilitiX/blob/new_logging-syslog/tests/include/unit_test.hpp

Reply:
fixed